README.md

```diff
@@ -1,24 +1,232 @@
 
-* Configuration
+[Air Reservation Management System Backend](https://github.com/Bestbynature/final-group-capstone.git) is just the end point of all the functionality on the Frontend app it is build on Ruby on rails using postgresql as a Database. Proper authorizations of users and authentication of user for creation reservations, flights and cancel them according to role 
 
-* Database creation
+## 🧪 Linters <a name="linters"></a>
+[![Linters](https://github.com/ernestmusong/Booking-app-frontend/actions/workflows/linters.yml/badge.svg)](https://github.com/ernestmusong/Booking-app-frontend/actions/workflows/linters.yml)
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+In this project we utilize our skills and knowledge to create this project
 
-* Deployment instructions
+<details>
+  <summary>Project language </summary>
+  <ul>
+    <li>JavaScript</li>
+    <li>Ruby on Rails</li>
+    <li>React and Redux</li>
+  </ul>
+</details>
 
-* ...
+<details>
+  <summary>FrameWork</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>DataBase</summary>
+  <ul>
+    <li><a href="postgresql.org/">Postgres</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Frontend and Backend FrameWork</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">ROR</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- *log in*
+- *Sign in*
+- *Responsive*
+- *Book Flight*
+- *Reserve Flight*
+- *Cancel Reservation*
+
+## 💻 API Documentation <a name="api-doc"></a>
+
+To get to the API Documentation, go to the link above after starting rails server
+
+```
+localhost:3000/api-docs
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project with the [Project Repo](https://github.com/Bestbynature/final-group-capstone.git) you need:
+
+- Node installed in your machine
+- Ruby installed in your machine
+- Ruby on Rails installed in your machine
+- Postgres installed and all password and user being setup
+- Any integrated terminal like 
+  - CMD `preferred`
+  - Bash`preferred`
+  - PowerShell
+  - etc
+
+### Setup
+
+for cloning follow this steps
+- go to green `code` button very top left corner of this Repository container
+- copy your `SSH` or `https` file and clone it in your desired folder 
+- or just download `ZIP` file
+- Create `.env` file in root directory and add this line there `port=8000` 
+
+*Note* for [Project repo](https://github.com/Bestbynature/final-group-capstone.git) follow there prerequisites
+### Install
```

Comment:
You need to put more in this section about getting the app set up like `bundle install` etc. If i were to follow these steps directly, the app would not run.

Make sure someone with barely any knowledge of rails can follow instructions clearly and run the application in their local environment